src/ldp/http/SimpleBodyParser.ts

```diff
@@ -0,0 +1,52 @@
+import { BodyParser } from './BodyParser';
+import { HttpRequest } from '../../server/HttpRequest';
+import { Quad } from 'rdf-js';
+import { QuadRepresentation } from '../representation/QuadRepresentation';
+import { RepresentationMetadata } from '../representation/RepresentationMetadata';
+import { StreamParser } from 'n3';
+import { TypedReadable } from '../../util/TypedReadable';
+import { UnsupportedMediaTypeHttpError } from '../../util/errors/UnsupportedMediaTypeHttpError';
+import 'jest-rdf';
```

Comment:
This import shouldn't be here?

Reply:
I accidentally added it to the src file instead of the test file, woops :D